### PR TITLE
Reorganize readme

### LINKED ISSUE #17: Duplicated creation the Language Translator & Natural Language Understanding service
From the Tokyo advocacy team:

There are two same kind of description for creating the Language Translator & Natural Language Understanding service. One is '2. Create language translation and natural language understanding service with IBM Cloud' and the other one is in '3. Run the server application in a container on IBM Cloud with  Kubernetes' with 'ibmcloud create service' command.

### LINKED ISSUE #18: Need to update Secret Names in watson-lang-trans.yml
From the Tokyo team:

There are no process description for updating Secret Names in watson-lang-trans.yml, but it is required

### PR DESCRIPTION
**Pull request by sanjeevghimire**
* reorganize readme
* added "secret names" instruction in readme
closes #17, closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,9 +107,12 @@ $ docker build -t registry.ng.bluemix.net/<ibmcloud_container_registry_namespace
 $ docker push registry.ng.bluemix.net/<ibmcloud_container_registry_namespace>/watsontesseract:1
 ```
 
-* Update the `image` in `watson-lang-trans.yml` with your image name
+* Update the `image` in `watson-lang-trans.yml` with your image name which in this case is `watsontesseract`.
```
Comment (scottdangelo) [inline README.md]: can `watsontesseract` be put in as a default?

File: README.md [hunk 2]
```diff
@@ -329,11 +338,9 @@ This error message is likely occurring because the service instance has not full
...
 * **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde) with all of our Code Pattern videos
 * **With Watson**: Want to take your Watson app to the next level? Looking to utilize Watson Brand assets? [Join the With Watson program](https://www.ibm.com/watson/with-watson/) to leverage exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
 
 # License
-This code pattern is licensed under the Apache Software License, Version 2.  Separate third party code objects invoked within this code pattern are licensed by their respective providers pursuant to their own separate licenses. Contributions are subject to the [Developer Certificate of Origin, Version 1.1 (DCO)](https://developercertificate.org/) and the [Apache Software License, Version 2](https://www.apache.org/licenses/LICENSE-2.0.txt).
-
-[Apache Software License (ASL) FAQ](https://www.apache.org/foundation/license-faq.html#WhatDoesItMEAN)
+[Apache 2.0](LICENSE)
```
Comment (scottdangelo) [inline README.md]: I think we added the stuff you are deleting on purpose (Laura Bennet and/or her team did this a couple months back). Maybe look at the git logs. This requirement came from legal.

File: README.md [hunk 3]
```diff
@@ -30,16 +30,19 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Node.js](https://nodejs.org/): An open-source JavaScript run-time environment for executing server-side JavaScript code.
 * [Tesseract OCR](https://github.com/tesseract-ocr/): An open-source Optical Character Recognition(OCR) engine.
 
+# Watch the Video
+
+TODO
+
 # Steps
 
 This Code Pattern contains several pieces. The Node.js server application running on IBM Cloud Kubernetes service communicates with the Tesseract OCR, Watson Language Translator and Watson Natural Language Understanding. Mobile application is built locally and run on the Android/iOS phone.
```
Comment (rhagarty) [inline README.md:40]: "and runs on the Android ..."

File: README.md (hunk 1 above)
Comment (rhagarty) [inline README.md]: suggestion - "Update the `image` in `watson-lang-trans.yml` with your image name `watsonesseract`.

File: README.md [hunk 4]
```diff
@@ -112,9 +107,12 @@ $ docker build -t registry.ng.bluemix.net/<ibmcloud_container_registry_namespace
 $ docker push registry.ng.bluemix.net/<ibmcloud_container_registry_namespace>/watsontesseract:1
 ```
 
-* Update the `image` in `watson-lang-trans.yml` with your image name
+* Update the `image` in `watson-lang-trans.yml` with your image name which in this case is `watsontesseract`.
 
+* Update the `namespace` in `watson-lang-trans.yml` with `<ibmcloud_container_registry_namespace>`
 
+* Update the `tag` in `watson-lang-trans.yml` with the tag number you created during docker push which in this case is `1`.
```
Comment (rhagarty) [inline README.md]: long sentence. Either put `,` after push, or put "which in ..." in parens

File: README.md [hunk 5]
```diff
@@ -127,9 +125,17 @@ $ kubectl apply -f watson-lang-trans.yml
 $ ibmcloud cs workers <cluster_name>
 ```
 
-## 4. Run the server application locally
+## 3. Run the server application locally using docker
 
-* Copy the `env.sample` to `.env` and replace the `IAM API` key and the `URL` that you got when you created the Watson language translation service. From terminal run:
+### 1. Create language translation and natural language understanding service with IBM Cloud
```
Comment (rhagarty) [inline README.md]: if these are sub-steps, make them 3a and 3b

**Review by rhagarty — CHANGES_REQUESTED** [review]
Also, if user can do step 2 OR step 3, should make that clearer.

**Review by scottdangelo — APPROVED** [review]
Will wait (a little while) for Rich to have a look before merging.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**